### ISSUE TITLE
Using GAT for regression

### ISSUE THREAD

**Issue by AdrianBZG** [Diego999/pyGAT#43]
Hi,

I would like to try this model out in a regression scenario, what sould be changed in order to do so? Is using a different loss function (one for regression) enough?

Thanks.

**Comment by tibe97** [comment]
I think you should also remove the softmax before returning the output in the forward method

**Comment by SuperBruceJia** [comment]
How about the labels w.r.t. the regression problem?
@Diego999 @tibe97 @AdrianBZG @pbloem @sh0416
